core/src/test/java/org/apache/iceberg/TestMetadataTableScans.java

```diff
@@ -606,7 +829,6 @@ public void testPartitionColumnNamedPartition() throws Exception {
     validateIncludesPartitionScan(tasksAndEq, 0);
   }
 
-
```

Comment:
Hello there

Reply:
Added back the space (it was an inconsistent 2 space between the tests)